Validation: Added Missing Descriptions (#240)
MicrosoftDocs/bingmaps-docs: BingMaps/sdk-native/map-control-concepts/map-polylines-and-polygons.md

---
title: Map Polylines and Polygons | Microsoft Docs

> description: Describes polylines and polygons and provides additional resources and examples to provide guidance for drawing with polylines and polygons.

Reviewer: ```suggestion
description: Describes polylines and polygons with a link to examples and additional information.
```
